commander/.eslintignore

```diff
@@ -1,4 +1,5 @@
 dist
 tmp
 coverage
-.eslintrc.js
\ No newline at end of file
+.eslintrc.js
+src/bootstrapping/templates/lisk-template-ts/templates/**/*
```

Comment:
we also should ignore this from typescript?

Reply:
Yes ignored it from there as well.